libs/dh/shared/util-apollo/src/lib/query.ts

```diff
@@ -150,7 +154,12 @@ export function query<TResult, TVariables extends OperationVariables>(
     refetch: (variables?: Partial<TVariables>) => {
       const result = firstValueFrom(result$.pipe(filter((result) => !result.loading)));
       const mergedVariables = { ...options$.value?.variables, ...variables } as TVariables;
-      options$.next({ ...options$.value, skip: false, variables: mergedVariables });
+      options$.next({
+        ...options$.value,
+        skip: false,
+        variables: mergedVariables,
+        fetchPolicy: 'network-only',
```

Comment:
Question: Is it a good idea to have this hard-coded here? It needs be we can provide default option values here libs/dh/shared/data-access-graphql/src/lib/dh-graphql.providers.ts

Reply:
It might be worth moving to `dh-graphql.providers.ts` but this will affect all `watchQuery`s, not only the ones executed through the `query` util function. Not sure if that can make Apollo fetch data in cases where it could just take it from cache. Besides that, the setting here is only used when calling `refetch` so its limited even more.

Reply:
Ah, if its only on refetch, I think its fine :-)